kano_world/functions.py

```diff
@@ -120,7 +120,9 @@ def login_register_data(data):
 
 
 def is_registered():
-    return 'kanoworld_id' in load_profile()
+    import mercury  # Lazy import to avoid dynamically linking with global import
+    kw = mercury.KanoWorld(kw_url)
```

Comment:
`load_data()` is called from the mercury constructor which is great. Propose we take this chance to remove this call from this function below:

 * https://github.com/KanoComputing/kano-profile/pull/475/files#diff-329c3d97fb7c5f2605957e44d77c9619R132